appserver/admingui/web/src/main/resources/configuration/virtualServers.jsf

```diff
@@ -40,87 +40,94 @@
+        />
 
 #include "/common/shared/listTableConfigButtons.inc"
 
-           <sun:tableRowGroup id="rowGroup1" selected="#{td.value.selected}" data={"$attribute{listOfRows}"} sourceVar="td">
-             <!afterCreate
-                 getClientId(component="$this{component}" clientId=>$page{tableRowGroupId});
-                />
+        <sun:tableRowGroup id="rowGroup1" selected="#{td.value.selected}" data={"$attribute{newListOfRows}"} sourceVar="td">
+            <!afterCreate
+                getClientId(component="$this{component}" clientId=>$page{tableRowGroupId});
+            />
             <sun:tableColumn headerText="$resource{i18n.common.SelectHeader}" selectId="select" rowHeader="$boolean{false}" id="col0">
-                    <sun:checkbox id="select"
-                        toolTip="$resource{i18n.common.select}"
-                        selected="#{td.value.selected}"
-                        selectedValue="$boolean{true}"
-                        onClick="setTimeout('admingui.table.changeOneTableButton(\\\\\'#{pageSession.topActionGroup}\\\\\', \\\\\'#{pageSession.tableId}\\\\\'); admingui.table.initAllRows(\\\\\'#{pageSession.tableId}\\\\\');', 0);"
-                    />
+                <sun:checkbox id="select"
+                              toolTip="$resource{i18n.common.select}"
+                              selected="#{td.value.selected}"
+                              selectedValue="$boolean{true}"
+                              onClick="setTimeout('admingui.table.changeOneTableButton(\\\\\'#{pageSession.topActionGroup}\\\\\', \\\\\'#{pageSession.tableId}\\\\\'); admingui.table.initAllRows(\\\\\'#{pageSession.tableId}\\\\\');', 0);"
+                              />
             </sun:tableColumn>
 
             <sun:tableColumn headerText="$resource{i18n.common.name.header}" sort="name" rowHeader="$boolean{true}" id="col1">
                 <sun:hyperlink id="link" text="#{td.value.name}"
-                    url="#{pageSession.editLink}&name=#{td.value.name}" />
+                               url="#{pageSession.editLink}&name=#{td.value.name}" />
             </sun:tableColumn>
 
-                 <sun:tableColumn headerText="$resource{i18n_web.vs.StateTableColumn}" sort="state" rowHeader="$boolean{false}" id="col2">
+            <sun:tableColumn headerText="$resource{i18n_web.vs.StateTableColumn}" sort="state" rowHeader="$boolean{false}" id="col2">
                 <staticText id="stateCol"  value="#{td.value.state}" />
             </sun:tableColumn>
 
             <sun:tableColumn headerText="$resource{i18n_web.vs.DefaultWebTableColumn}"  sort="defaultWebModule" rowHeader="$boolean{false}" id="col3">
                 <staticText id="stateCol"  value="#{td.value.defaultWebModule}" />
             </sun:tableColumn>
+
+            <sun:tableColumn headerText="$resource{i18n_web.vs.ContextRootTableColumn" sort="contextRoot" rowHeader="$boolean{false}" id="col4">
```

Comment:
$resource{i18n_web.vs.ContextRootTableColumn isn't being found for me, is the wrong resource file being used?
![image](https://user-images.githubusercontent.com/12214116/27631807-a2fabeae-5bf1-11e7-8616-bbda853a550e.png)